<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoTag</class>
 <widget class="QWidget" name="VideoTag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start|Cont.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Stop">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="CQtOpenCVViewerGl" name="MainV" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>250</width>
     <height>190</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Pause">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="CQtOpenCVViewerGl" name="TagStream" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="TagList">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddTag">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Tag</string>
   </property>
  </widget>
  <zorder>TagList</zorder>
  <zorder>Start</zorder>
  <zorder>Stop</zorder>
  <zorder>MainV</zorder>
  <zorder>Pause</zorder>
  <zorder>TagStream</zorder>
  <zorder>AddTag</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CQtOpenCVViewerGl</class>
   <extends>QWidget</extends>
   <header>cqtopencvviewergl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
